source/draw/gpu/pc/Window.ooc

```diff
@@ -23,31 +23,32 @@ use ooc-base
 import X11/X11Window
 import X11/include/x11
 
-Window: class extends OpenGLES3Context {
+Window: class extends GpuSurface {
 	_native: NativeWindow
 	_monochromeToBgra: OpenGLES3MapMonochromeToBgra
 	_bgrToBgra: OpenGLES3MapBgrToBgra
 	_bgraToBgra: OpenGLES3MapBgra
 	_yuvPlanarToBgra: OpenGLES3MapYuvPlanarToBgra
 	_yuvSemiplanarToBgra, _yuvSemiplanarToBgraTransform: OpenGLES3MapYuvSemiplanarToBgra
+	context ::= this _context as OpenGLES3Context
+	size ::= this _size
 
-	size: IntSize2D { get set }
-
-	init: /* internal */ func (=size, title: String) {
+	init: /* internal */ func (size: IntSize2D, title: String) {
 		this _native = X11Window new(size width, size height, title)
-		super(this _native)
+		super(size, OpenGLES3Context new(this _native), FloatTransform2D createScaling(2.0f / size width, -2.0f / size height))
```

Comment:
This is the same transform used in the `GpuCanvas` constructor. Can it be made a static property somewhere that both classes use, instead of creating it in two places?